HackYourFuture-CPH/Guide-IT
# Career result

# Description

we created Full stack Career Result

Fixes # (issue)
#136 

# How to test?

test this by "npm run dev" and check here http://localhost:3000/quiz-results/userid in numbers like 1 or 2  


# Checklist

- [x ] I have performed a self-review of my own code
- [ x] This PR is ready to be merged and not breaking any other functionality
